### DISCUSSION TITLE
Test cases are failing in inquiry branch release-6.0.0

### DISCUSSION THREAD

**Discussion by githubpiyush** [Sunbird-inQuiry/Community#85] [Q&A] [answered]
@krgauraw 

I have pulled the latest code from inquiry 6.0.0 branch but when I tried below command I am getting error in HierarchyManagerTest test case.
You can see attached screenshot of the error.

`mvn clean install`

![image](https://github.com/Sunbird-inQuiry/Community/assets/35293206/81ea5ca8-cd6b-4b25-b807-8b9ebe0382e4)

can you help check the same ?
If it is specific to me only then how can I resolve this issue?
 
CC :- @aks30

**Comment by krgauraw** [comment]
@githubpiyush as you can see the test case is not able to create the directory "/data/logs". So please create it manually under root directory of the system.

Hope this helps!

cc: @aks30

**Reply by githubpiyush** [reply]
@krgauraw I tried that and still getting the same error.

Also can you help me on the following parts :-

1. How can we restrict users to use certain API at OPA level?
2. How to use system update to update any questionSet?(it's a forced update which does not require versionKey)

**Reply by sajeshkayyath** [reply]
@githubpiyush - once you create the directory, change the ownership. Linux command sample is as below,

```shell
sudo chown -R $USER /data
``` 

This will fix the issue.

**Reply by githubpiyush** [reply]
@sajeshkayyath 
I have tried to give thr write permission to data folder as well as the root folder but still getting same error PFA screenshot for the same

![image](https://github.com/Sunbird-inQuiry/Community/assets/35293206/af916c1d-b7db-4ddf-a99a-3865d69a332b)
![image](https://github.com/Sunbird-inQuiry/Community/assets/35293206/b7b40d7c-0873-455e-94b9-7dcb275a6dde)
![image](https://github.com/Sunbird-inQuiry/Community/assets/35293206/52c10fcd-72af-4025-b8d2-89e624de9de9)
![image](https://github.com/Sunbird-inQuiry/Community/assets/35293206/3ed62cb6-7b8d-4979-8576-31a49be891cb)

**Reply by krgauraw** [reply, marked as answer]
@githubpiyush can you please try running below command from your terminal and then try the test cases.

sudo chmod -R 777 /data/logs
